Orbital-Docs/_pages/docs/designer_style_guide.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: single
+title: Orbital Designer Style Guide
+permalink: /docs/designer_style_guide/
+sidebar:
+  nav: 'docs'
+---
+
+# Orbital Designer Style Guide
+
+Much of our style guide pulls directly from [Angulars Style Guide](https://angular.io/guide/styleguide). When in doubt , their documentation is a good place to look.
+
+##Guiding Principle: Consistency is kind.
```

Comment:
missing space after the double #

Reply:
Awesome, also updated kind to king and added a newline underneath